samples/boards/intel_s1000_crb/audio/audio_app.rst

```diff
@@ -25,22 +27,29 @@ The first 2 channels are buffered and forwarded to the host.
 Simultaneously, 2 channels of audio from the host are buffered and forwarded
 to the codec.
 This bidirectional forwarding of audio demonstrates a 2-way audio stream
-between S1000's microphones/audio line-out and the host.
+between the Sue Creek board's microphones/audio line-out and the host.
```

Comment:
Legal names can't be abbreviated, so this should be ``the Intel® Sue Creek S1000 board's``

Reply:
recheck